packages/kbn-optimizer/limits.yml

```diff
@@ -126,6 +126,7 @@ pageLoadAssetSize:
   savedSearch: 16225
   screenshotMode: 17856
   screenshotting: 22870
+  searchConnectors: 30000
```

Comment:
Is it necessary for this to be 300kb (I believe this is a typo currently)?  We aim for most plugins to have sync bundle sizes < 100kb.

Reply:
I'll take a look at this again, my instinct was that it's blowing up because it's essentially just a collection of images and that increases bundle size, but that may be wrong.

Reply:
Yeah oops, it was pulling in the full `kbn/search-connectors` package because it was importing a const from there, and that was blowing up the bundle size. I moved that const to this plugin and changed some references elsewhere, it should be under 30kb now (so this typoed limit actually makes sense now).

Reply:
Awesome, thank you!